pkg/util/shortid_generator.go

```diff
@@ -0,0 +1,24 @@
+package util
+
+import (
+	"github.com/teris-io/shortid"
+)
+
+func init() {
+	gen, _ := shortid.New(1, shortid.DefaultABC, 1)
+	shortid.SetDefault(gen)
+
+}
+
+// GenerateShortUid generates a short unique identifier.
+func GenerateShortUid() (uid string, err error) {
+	if uid, err = shortid.Generate(); err != nil {
+		if uid, err = shortid.Generate(); err != nil {
+			if uid, err = shortid.Generate(); err != nil {
+				return "", err
+			}
+		}
+	}
+
```

Comment:
Why is this necessary? I don't see anything in `shortid.Generate` that is expected to fail, and the library itself provides `shortid.MustGenerate` which panics on failure, so I don't think the author expects it to fail either

Reply:
That's a really good question :) Late coding and not thinking about what possible errors the lib could generate. I'll fix. Thanks for the extra eye :+1: